GAPIT.md

```diff
@@ -0,0 +1,1752 @@
+**Note**: Autogenerated documentation, do not modify!
```

Comment:
This file should not be checked into master, but the gh-pages branch, which powers gapid.dev. We should avoid checking in generated source into master. The only generated files left that are checked in are third party config files.